packages/fixie/src/main.ts

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env node
+
+/**
+ * This is a command-line tool to interact with the Fixie platform.
+ */
+
+import { program } from 'commander';
+import terminal from 'terminal-kit';
+import { FixieClient } from './client.js';
+
+const { terminal: term } = terminal;
+
+program
+  .name('fixie')
+  .version('1.0.0')
+  .description('A command-line client to the Fixie AI platform.')
+  .option('-u, --url <string>', 'URL of the Fixie API endpoint', 'https://app.fixie.ai');
+
+program.command('user').action(async () => {
+  const client = await FixieClient.Create(program.opts().url);
+  const userInfo = await client.userInfo();
+  term('You are logged in as: \n').green(JSON.stringify(userInfo, null, 2));
+});
+
+const corpus = program.command('corpus').description('Corpus related commands');
+
+corpus
+  .command('list')
+  .description('List all corpora.')
+  .action(async () => {
+    const client = await FixieClient.Create(program.opts().url);
+    const result = await client.listCorpora();
+    term('Corpora: \n').green(JSON.stringify(result, null, 2));
+  });
+
+corpus
+  .command('get <corpusId>')
+  .description('Get information about a corpus.')
+  .action(async (corpusId: string) => {
+    const client = await FixieClient.Create(program.opts().url);
+    const result = await client.getCorpus(corpusId);
+    term(`Corpus [${corpusId}]: \n`).green(JSON.stringify(result, null, 2));
+  });
+
+corpus
+  .command('create [corpusName]')
+  .description('Create a corpus.')
+  .action(async (corpusName?: string) => {
+    const client = await FixieClient.Create(program.opts().url);
+    const result = await client.createCorpus(corpusName);
+    term('Created corpus: \n').green(JSON.stringify(result, null, 2));
+  });
+
+corpus
+  .command('query <corpusId> <query>')
+  .description('Query a given corpus.')
+  .action(async (corpusId: string, query: string) => {
+    const client = await FixieClient.Create(program.opts().url);
+    const result = await client.queryCorpus(corpusId, query);
+    term('Query result: \n').green(JSON.stringify(result, null, 2));
```

Comment:
Hmm. When I get a JSON output, I like to be able to pipe it to `jq` for further processing. However, with the current form, I have to do things like `tail` and `sed -E '1d; s/\x1B\[[0-9;]*[mG]//'` to make this `jq`-able.

It would be nice to just dump JSON.

Reply:
If you're attached to the nicer output, maybe we could have a `--format json` flag?

Reply:
OK, I removed the extra lines at the top, so everything can be piped to jq, and added a `--raw` option which disables pretty-printing. I'd rather have pretty-printing on by default because I *think* most people running this tool will want that rather than hunt for an option...